### PR TITLE
Harbor SIG-Security Assessment

### PR DESCRIPTION
**Pull request by michmike**
Signed-off-by: Michael Michael <michmike@cs.stanford.edu>

### CODE REVIEW

File: assessments/projects/Harbor/self-assessment.md [hunk 1]
```diff
@@ -0,0 +1,1166 @@
...
+   </td>
+  </tr>
+</table>
+
+# Overview
+
+The Harbor project is an enterprise-grade container registry that enables enterprises to assess the security posture of container images and enforce that images are compliant with being free from vulnerabilities as defined in the configured vulnerability databases.
+
+
+    Vulnerabilities are only detected in an artifact or container image as the result of static analysis. This static analysis is performed by a scanner like CoreOS’s Clair or Aqua’s Trivy. If the vulnerabilities are already published and the scanner can pick them up, images are being analyzed against those vulnerabilities. In the case of Zero Day CVEs, these take time to be published in [vulnerability databases](https://github.com/quay/clair/blob/master/Documentation/drivers-and-data-sources.md#data-sources-for-the-built-in-drivers) and may take hours or days to be picked up by Trivy or Clair. Each scanner has their own vulnerability databases that they use for scanning.
```
Comment (JustinCappos) [inline assessments/projects/Harbor/self-assessment.md:103]: This indentation causes weird formatting issues.  Can you fix it?

File: assessments/projects/Harbor/self-assessment.md [hunk 2]
```diff
@@ -0,0 +1,1166 @@
...
+*   On the next release of Harbor v2.0, we will also be removing the ability for a user to query a password reset to prevent accidentally disclosing information when used maliciously for phishing.
+*   Harbor 2.0 will also allow for enforcing TLS communication between all internal components as described above.
+
+
+## Project Compliance
+
+Harbor does not meet any compliance standards today. However, as we interact with more service providers that are putting Harbor in production (example by OVH Cloud [here](https://www.ovh.com/blog/managing-harbor-at-cloud-scale-the-story-behind-harbor-kubernetes-operator/)) and delivering it as a service, we understand there may be a need to acquire some of these certifications. We will continue exploring this vertical and their needs of Harbor and prioritize any requests. Currently, there haven’t been any regarding compliance certifications.
+
+
+    There is only 1 public reference of a service provider today. However we are working privately with a few more service providers that don't want to be named publicly yet. There are also some enterprises that are internally acting as service providers and have the same use cases.
```
Comment (JustinCappos) [inline assessments/projects/Harbor/self-assessment.md:647]: Indentation is causing a markdown issue.

**Comment by michmike** [comment]
> A few little problems with markdown. I didn't review in detail.

they were notes that were put on a block. i updated the type of note to make it easier to read.

**Comment by michmike** [comment]
Fixes https://github.com/cncf/sig-security/issues/369

**Comment by lumjjb** [comment]
Minor nit, can the folder be renamed to lowercase to follow convention of assessment directory structure. Thanks!

**Comment by michmike** [comment]
> Minor nit, can the folder be renamed to lowercase to follow convention of assessment directory structure. Thanks!

thanks. I saw that, but i opted to keep it uppercase since Harbor is the actual name of our product and a brand.

**Pull request merged**